src/main/java/salesforce/entities/constants/ContractConstant.java

```diff
@@ -0,0 +1,23 @@
+/*
+ * Copyright (c) 2019 Jalasoft.
+ *
+ * This software is the confidential and proprietary information of Jalasoft.
+ * ("Confidential Information"). You shall not disclose such Confidential
+ * Information and shall use it only in accordance with the terms of the
+ * license agreement you entered into with Jalasoft.
+ */
+
+package salesforce.entities.constants;
+
+/**
+ * Manages ContractConstant instance.
+ *
+ * @author Alan Escalera.
+ * @version 1.0 29 March 2020.
+ */
+public class ContractConstant {
```

Comment:
if this class is only to declare constant variable, it should be **public final class ...**

Reply:
x2